src/contracts/identity.ts

```diff
@@ -130,3 +154,15 @@ export const isAuthorizedTo = async (
 
   return Identity__factory.connect(contractAddress, provider).isAuthorizedTo(address, permission, blockNumber);
 };
+
+const getBeaconAddress = async () => {
```

Comment:
question for me: when do we add the return type in typescript

Reply:
@enddynayn I believe it should have a return type.

Reply:
Thank you. Added.